src/lib.rs

```diff
@@ -1200,6 +1200,14 @@ impl<T> Slab<T> {
     }
 }
 
+impl<T> Extend<T> for Slab<T> {
+    fn extend<I: IntoIterator<Item = T>>(&mut self, iter: I) {
+        for value in iter.into_iter() {
+            self.insert(value);
+        }
+    }
+}
```

Comment:
Inserting values into a slab without saving the resulting indexes is usually not useful. Do you have a use-case for this?

Reply:
I was thinking about that, I personally have a use case where the resulting indices aren't used, but it might be good to not use the trait at all and just have a function that returns `Vec<usize>` or an iterator like that instead?

Reply:
Something like this might work:

```rs
pub fn extend<I: IntoIterator<Item = T>>(&mut self, iter: I) -> Vec<usize> {
    iter.into_iter().map(|x| self.insert(x)).collect()
}

```

Reply:
Actually, I think I prefer the original version. A `Vec<usize>` will often not be the form that the user wants the indexes in.

I think it's okay to just say that users who want the indexes can iterate it themselves.

Reply:
What about `-> impl Iterator<Item=usize>`?

Reply:
That could make sense, but it would be a larger change. I don't think we can do it without a custom iterator.

Not having a return value allows us to follow the existing trait.